src/GraphQl/State/Provider/ReadProvider.php

```diff
@@ -68,6 +69,10 @@ public function provide(Operation $operation, array $uriVariables = [], array $c
                 $item = null;
             }
 
+            if ($operation instanceof Query && null === $item) {
```

Comment:
shouldn't it be at line 86 that we check if the `$item` is not null? Also can you add a non regression test?

Reply:
I put it above because it looks like condition `!\is_object($item)` on line 86 is related to Subscription and Mutation operations logic and `$item` should be checked on null before it.

Reply:
As for test - I did it but unfortunately `ApiPlatform\Exception\ItemNotFoundException` is outside of GraphQL directory and I cannot mock `IriConvertorInterface` to throw this exception. Used `class_alias` to make it work, not sure if it's OK.